BL_table/BL_table.Rmd

```diff
@@ -0,0 +1,259 @@
+
+**Important note: in RCT's, differences between randomized groups should not be tested for statistical significance**. Indeed, some imbalance is unavoidable owing to chance, especially if many characteristics are considered (@Palesch)
+
+
+```{r gg-oz-gapminder, echo=FALSE, fig.cap="Figure 1: Basic baseline structure and analysis-specific considerations affecting columns, rows, and cells. Figure from (@HAYESLARSON2019125)", message=FALSE}
+library(knitr)
+library(jpeg)
+img1_path <- "Figure/fig1.jpg"
+img1 <- readJPEG(img1_path, native = TRUE)
+include_graphics(img1_path)
+```
+
+# How to program a baseline table
+
+Many developers have published tools for building baselines tables.
+When choosing one please consider the flexibility of the tool.
+Indeed requirements for the contents and formatting of baseline table may vary depending on the project, the authors, the target journal... Here we present few (very flexible) options for R and stata.
+
+## In R
+
+### Data
+
+```{r eval=TRUE, include=TRUE}
+library(dplyr)
+library(Hmisc)
+data(mtcars)
+mtcars$am_f <- factor(mtcars$am, 0:1, c("Manual", "Automatic")) 
+mtcars$vs_f <- factor(mtcars$vs, 0:1, c("V", "Straight")) 
+d <- mtcars %>%  select(mpg, cyl, am_f,vs_f)
+
+Hmisc::label(d$mpg) <- "Miles per gallon"
+Hmisc::label(d$cyl) <- "Cylinders"
+Hmisc::label(d$am_f) <- "Transmission"
+Hmisc::label(d$vs_f) <- "Engine"
+
+```
+
+### Using gtsummary
+
+```{r}
+#Make a table stratify the table by transmission
+library(gtsummary)
+table1=d %>%  tbl_summary(by = am_f)
+```
+
+```{r table1a_fig, results="hide", include=FALSE}
+a=as_gt(table1)
+gt::gtsave(a, "Table/table1a.png")
+```
+<img src='Table/table1a.png' align="middle" width="60%" >
+
+This is the basic usage; defaults options may be customized.
+
+- Variable types are automatically detected so that appropriate descriptive statistics are calculated.
+- Label attributes from the data set are automatically printed.
+- Missing values are listed as "Unknown" in the table.
+- Variable levels are indented and footnotes are added.
+
+Once produced gtsummary tables can be converted your favorite format (e.g. html/pdf/word). For more information see [here](https://www.danieldsjoberg.com/gtsummary/articles/rmarkdown.html))
+
+```{r}
+# declare cylinders as a continuous variable, for this variable calculate the mean and sd value, add an overall column, change the missing text
+table1=d %>%
+        tbl_summary(by = am_f, 
+                    type = list(cyl ~ 'continuous'),
+                    statistic = list(cyl ~ "{mean} ({sd})"),
+                    missing_text = "Missing") %>%
+        add_overall()
+
+```
+
+```{r table1_fig, results="hide", include=FALSE}
+a=as_gt(table1)
+gt::gtsave(a, "Table/table1b.png")
+```
+<img src='Table/table1b.png' align="center" width="80%">
+
+
+
+
+
+
+If needed groups may be compare using statistical tests
```

Comment:
should we really introduce this functionality? Because this documentation is about table 1, where we wouldn't expect p-values, maybe we can leave this one out. This could be used in another vignette about comparison table (where groups are not just described on their baseline characteristics but on relevant variable in simple analyses).